qasources.md

```diff
@@ -0,0 +1,50 @@
+## Main Endpoint
+**GET /api/integration/qasources**
+
+Provide access to the Quality Assurance sources. It returns the list of the Quality Assurance sources.
+
+```json
+[
+
+  {
+    id: "openaire",
+    type: "qasource",
+    totalEvents: "33"
+  },
+  {
+    id: "another-source",
+    type: "qasource",
+    lastEvent: "2020/10/09 10:11 UTC",
+    totalEvents: "21"
+  },
+  ...
+]
+```
+Attributes:
+* lastEvent: the date of the last update from the specific Quality Assurance source
+* totalEvents: the total number of suggestions provided by the Quality Assurance source
```

Comment:
Please replace "suggestions" with "Quality Assurance Events" or just Events.  We may also want to mention here that events can be found using the "qaevents" endpoint, as that'll help people find the related endpoints.